zuul.d/jobs.yaml

```diff
@@ -1787,6 +1787,69 @@
     vars:
       ansible_collections_repo: github.com/ansible-collections/frr.frr
 
+- job:
+    name: ansible-test-sanity-yang
+    parent: ansible-test-sanity-base
+    required-projects:
+      - name: github.com/ansible-collections/community.yang
+    timeout: 3600
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/community.yang
+
+- job:
+    name: ansible-test-units-yang-python27
+    parent: ansible-test-units-base-python27
+    required-projects:
+      - name: github.com/ansible-collections/community.yang
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/community.yang
+
+- job:
+    name: ansible-test-units-yang-python35
+    parent: ansible-test-units-base-python35
+    required-projects:
+      - name: github.com/ansible-collections/community.yang
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/community.yang
+
+- job:
+    name: ansible-test-units-yang-python36
+    parent: ansible-test-units-base-python36
+    required-projects:
+      - name: github.com/ansible-collections/community.yang
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/community.yang
+
+- job:
+    name: ansible-test-units-yang-python37
+    parent: ansible-test-units-base-python37
+    required-projects:
+      - name: github.com/ansible-collections/community.yang
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/community.yang
+
+- job:
+    name: ansible-test-units-yang-python38
+    parent: ansible-test-units-base-python38
+    required-projects:
+      - name: github.com/ansible-collections/community.yang
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/community.yang
+
+- job:
+    name: ansible-test-network-integration-community-yang-iosxr-netconf-python36
+    parent: ansible-test-network-integration-iosxr-python36
+    vars:
+      ansible_collections_repo: github.com/ansible-collections/community.yang
+      ansible_test_integration_targets: "yang_.*"
```

Comment:
The prefix pattern is valid in case the collection has test for multiple platforms or in case of stable-2.9 ansible. Since all the test are contained in yang collection this prefix is not required. This can be replaced with `ansible_test_integration_targets: ""`

Reply:
To run all the test within yang collection a better way is to add `yang` group in iosxr and junos inventory file template refer
1) https://github.com/ansible/ansible-zuul-jobs/blob/1586f4e2aad2a211c7c7b196a937fd26d8c86abd/roles/ansible-test-inventory/templates/inventory-iosxr.j2#L7 

2) https://github.com/ansible/ansible-zuul-jobs/blob/1586f4e2aad2a211c7c7b196a937fd26d8c86abd/roles/ansible-test-inventory/templates/inventory-junos.j2#L7

By adding the group aliases in the yang collection integration test
Refer https://github.com/ansible-collections/community.yang/blob/af0f2da3faf465742274d99aca969954e2d3b79c/tests/integration/targets/fetch/aliases#L1
renaming test folder to prefix with `yang_*` is not required